### PR TITLE
Add support for pagination on static paths/props

### PR DESCRIPTION
**Pull request by ionutpasca**

### CODE REVIEW

File: examples/uidl-samples/cms-project.json [hunk 1]
```diff
@@ -0,0 +1,1324 @@
...
+                },
+                "route": {
+                  "type": "static",
+                  "value": "api/books"
+                }
+              }
+            }
+          },
+          {
+            "value": "books/Books1",
```
Comment (elizamaria) [inline examples/uidl-samples/cms-project.json]: This should be books/Books

File: examples/uidl-samples/cms-project.json [hunk 2]
```diff
@@ -0,0 +1,1324 @@
...
+                {
+                  "property": "og:title",
+                  "content": "Soggy Insubstantial Octopus"
+                }
+              ]
+            },
+            "pageOptions": {}
+          },
+          {
+            "value": "/books/Books",
```
Comment (elizamaria) [inline examples/uidl-samples/cms-project.json]: This should be books/Book1

File: examples/uidl-samples/cms-project.json [hunk 3]
```diff
@@ -0,0 +1,1324 @@
...
+                "authToken": {
+                  "type": "env",
+                  "value": "CMS_ACCESS_TOKEN"
+                },
+                "urlParams": {
+                  "populate": {
+                    "type": "static",
+                    "content": "*"
+                  },
+                  "pagination": {
```
Comment (elizamaria) [inline examples/uidl-samples/cms-project.json:683]: Can't we skip sending this? We already send all pagination related info in the "pagination" object, and we know the pageSize

**Review by elizamaria — APPROVED** [review]
Good go to

File: examples/uidl-samples/cms-project.json (hunk 1 above)
Comment (ionutpasca) [reply examples/uidl-samples/cms-project.json]: Make a change

File: examples/uidl-samples/cms-project.json (hunk 2 above)
Comment (ionutpasca) [reply examples/uidl-samples/cms-project.json]: Made a change

File: examples/uidl-samples/cms-project.json (hunk 3 above)
Comment (ionutpasca) [reply examples/uidl-samples/cms-project.json:683]: We know the pagination stuff, but this is a more general implementation. This is inside the `initialPropsResource - urlParams`, that can be anything...  Whenever we are building the initial props, we don't really want to use information from outside of this scope. Does it make sense?

**Pull request merged**